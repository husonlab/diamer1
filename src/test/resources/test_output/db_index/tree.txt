parent id	node id	rank	label	kmers in database
	0	no rank	root	2
0	1	1 from root	rank1	167
0	2	1 from root	rank2	284
1	4	2 from root	organism2	449
1	5	2 from root	organism3	328
2	3	2 from root	organism1	92
