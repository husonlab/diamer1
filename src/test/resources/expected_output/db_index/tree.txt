parent id	node id	rank	label	kmers in database
	0	no rank	root	2
0	1	1 from root		167
0	2	1 from root		284
1	4	2 from root		449
1	5	2 from root		328
2	3	2 from root	organism1	92
